library-source/fp/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
+    kotlin("jvm")
+}
+
+dependencies {
+    api("io.arrow-kt:arrow-core:0.11.0")
```

Comment:
For the moment you only need the core dependecy.

Reply:
ok